phylanx/util/instrumentation/performance_data.hpp

```diff
@@ -0,0 +1,33 @@
+//  Copyright (c) 2018 Parsa Amini
+//  Copyright (c) 2018 Hartmut Kaiser
+//
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#if !defined(PHYLANX_UTIL_PERFORMANCE_DATA)
+#define PHYLANX_UTIL_PERFORMANCE_DATA
+
+#include <phylanx/config.hpp>
+
+#include <hpx/runtime/find_here.hpp>
+
+#include <map>
+#include <vector>
+
+
+namespace phylanx { namespace util
+{
+    // Retrieve performance counter data for selected primitives
+    //     primitive_instances:
+    //         The primitives for which performance counter data is required
+    //     locality_id:
+    //         The locality the performance counter data is going to be queried from
+    //     counter_name_last_parts:
+    //         A vector containing the last part of the performance counter
+    //         names. e.g. std::vector{ "count/eval", "count/direct_eval" }
+    PHYLANX_EXPORT std::map<std::string, std::vector<std::int64_t>>
```

Comment:
Could you turn this comment into a proper doxygen formatted one, please?

Reply:
Please check again.